ArticleTemplates/assets/js/bootstraps/gallery.js

```diff
@@ -1,120 +1,114 @@
-import { debounce } from 'modules/util';
-
-let galleryProps;
-let galleryRows;
-
-const PROPS = {
-        0 :{
-            imagesPerRow: 3
-        },
-        650: {
-            imagesPerRow: 4
-        },
-        1200: {
-            imagesPerRow: 5
-        }
-    };
+import Hammer from 'hammerjs';
 
 function init() {
-    buildGallery();
-    window.addEventListener('resize', debounce(buildGallery, 100));
-}
+    lazyLoadImages();
+    var galleryImages = document.querySelectorAll('.touch-gallery__images');
+    galleryImages.forEach(function(galleryImage) {
+        var hammerSettings = (GU.opts.platform === 'ios' ? { touchAction: 'auto' } : {})
```

Comment:
`hammerSettings` will be the the same for each `galleryImage` so you can move this declaration outside the `forEach` and only run this once.